src/main/java/org/beanpod/switchboard/controller/LogController.java

```diff
@@ -24,4 +31,15 @@
   public ResponseEntity<List<LogModel>> retrieveDeviceLogs(@PathVariable String serialNumber) {
     return ResponseEntity.ok(logDao.getDeviceLogs(serialNumber));
   }
+
+  @Override
+  public ResponseEntity<LogModel> createLog(@Valid CreateLogRequest createLogRequest) {
+    return Optional.of(createLogRequest)
+        .map(logMapper::toLogDto)
+        .map(logDao::createLog)
+        .map(logMapper::toLogEntity)
+        .map(logMapper::toLogModel)
```

Comment:
why do we log to entity and then to model cant we directly toLogModel

Reply:
We want to go from LogDto to LogModel, but toLogModel goes from LogEntity to LogModel

Reply:
why no create a new method?

LogModel toLogModel(LogDto x)

Reply:
Good suggestion to remove middle mappings. I have updated the code accordingly.